spynnaker/pyNN/models/neuron/master_pop_table.py

```diff
@@ -118,21 +159,20 @@ def get_master_population_table_size(self, in_edges):
                 # TODO: Fix this to be more accurate!
```

Comment:
the docs for this method has a vertex slice which doesnt exist in this methods params.......

Reply:
Removed